### DISCUSSION TITLE
failed compatiblity check issue in amd raedon r5

### DISCUSSION THREAD

**Discussion by razrexe** [Lolliedieb/lolMiner-releases#305] [Q&A]
When I configured lolminer on my pc, there is this prompt:-

New job received: 0xc970a8 Epoch: 397 Target: 0000000112e0be82
Start Mining...
All devices deselected or failed compatibility check. Closing lolMinerPress any key to continue . . .

Anybody else faced this issue with AMD chips?
can somebody tell any workaround

**Comment by LakkaSleep** [comment]
Hi razrese,

I'm having the same issue. Did you happen to find a resolution?

**Comment by LakkaSleep** [comment]
hey razrese,

I was able t fix my own problem. lol  I had installed the standard drivers from the Gigabyte website but those drives don't work as they are not the true "AMD" drivers and i don't believe show up correctly. On the same Gigabyte support webpage for your care scroll down and you should see a link in Blue that says "Please click here to download the latest driver version from GPU manufacturer of your graphics card" . Download and install this drivers. That should do the trick and worked for me. Prior to that i had spent at least 4 hours trying to get it to work.

Hope this works for you.
